lib/dash/mpd_utils.js

```diff
@@ -142,8 +142,8 @@ shaka.dash.MpdUtils = class {
     const timeline = [];
     let lastEndTime = -unscaledPresentationTimeOffset;
 
-    const enumerate = (it) => shaka.util.Iterables.enumerate(it);
-    for (const {item: timePoint, next} of enumerate(timePoints)) {
+    for (let i = 0; i < timePoints.length; ++i) {
```

Comment:
@avelad you missed the second enum loop below for the repetitions - this allows removing the `Iterables` dependency completely here as well.

Reply:
Done!